roles/ansible-test/tasks/init_single_mode.yaml

```diff
@@ -1,5 +1,12 @@
 ---
-- name: Setup location of project
+- name: Setup location of project - integration tests
   set_fact:
     _test_location: "{{ ansible_test_location }}"
     _test_cfg_location: "{{ ansible_test_location }}/test/integration/{{ ansible_test_test_command }}.cfg"
+  when: ansible_test_test_command == 'network-integration'
```

Comment:
we can drop this when command, as we have both 'integration' and 'network-integration' commands